### PR TITLE
Several character rendering erros: "z.u00A0B." instead of "z. B."

### PR DESCRIPTION
**Pull request by freshking**
## Description

Fix no-break space character. Should be `\U00A0` instead of `\u00A0`.

## Link to Jira

https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8240

### CODE REVIEW

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -2452,9 +2452,9 @@ Die Angaben werden statistisch ausgewertet, sie werden nicht zu einem Profil ges
...
+"ErrorReport_Legal_DetailedInformation_Content2" = "Nachdem die Echtheit Ihrer App geprüft wurde, wird der Fehlerbericht über eine gesicherte Verbindung an das RKI übermittelt. Die Fehlerberichte werden nur zur Fehleranalyse und Fehlerbehebung im Rahmen zukünftiger Updates der App genutzt. Nur Mitarbeiter des technischen Support können auf die Fehlerberichte zugreifen. Die Fehlerberichte enthalten eine Vielzahl von Statusmeldungen und Ereignissen, die in der App ausgelöst wurden, sie enthalten aber keine Hinweise anhand derer das RKI auf Ihre Identität schließen kann. Erst wenn Sie die Fehlerbericht-ID im Zusammenhang mit weiteren Mitteilungen nennen, kann ein Zusammenhang zwischen der Mitteilung (und z.\U00A0B. Ihrem dort enthaltenen Namen) und den im Fehlerbericht enthaltenen Angaben (z. B. technische Meldungen zur Berechnung im Rahmen der Risiko-Ermittlung, in der App angezeigte Informationen und durchlaufene Schritte, aber ggf. auch einem abgerufenen Testergebnis und ggf. im Rahmen der Warnung geteilter Zufalls-IDs) hergestellt werden.";
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:2457]: Whats up with these lines? Do they contain non-breaking spaces as well?

Comment (freshking) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:2457]: Yes

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Comment by freshking** [comment]
Have to change base branch

**Pull request closed without merging**